### PR TITLE
Add scan time support in soda-spark

### PR DESCRIPTION
**Pull request by shannywu**
pass scan time with scan configs when create scans

### CODE REVIEW

File: src/sodaspark/scan.py [hunk 1]
```diff
@@ -264,19 +265,26 @@ def create_scan(
     variables: variables to be substituted in scan yml
     soda_server_client : Optional[SodaServerClient] (default : None)
         A soda server client.
+    time: timestamp date in ISO8601 format (defaul: None)
```
Comment (JCZuurmond) [inline src/sodaspark/scan.py:268]: Could you write the same as in the function below:

```
<parameter name> : <parameter type> (default: <default value)
    <parameter description
```

And add to the description what happens if `None` -> use `now...`

File: src/sodaspark/scan.py [hunk 2]
```diff
@@ -264,19 +265,26 @@ def create_scan(
...
 
     Returns
     -------
     out : Scan
         The scan.
     """
+
+    if not time:
```
Comment (JCZuurmond) [inline src/sodaspark/scan.py:276]: I prefer:

``` python
time = time or dt.datetime.now(tz=dt.timezone.utc).isoformat(timespec="seconds")
```

**Review by JCZuurmond — COMMENTED** [review]
Thanks for your contribution @shannywu! Could you add a line to the `CHANGELOG.md` explaining your change? Also, I have some minor feedback, other than that LGTM.

**Comment by shannywu** [comment]
Hi @JCZuurmond ,

Thanks for the quick review. I update the code per your suggestions. Please have a look.

**Comment by vijaykiran** [comment]
Hi @shannywu   - I'll take care of the conflicts .. sorry, I didn't see your changes before I made the other PR.

**Pull request merged**